app/javascript/components/Results/Jobs/Jobs.tsx

```diff
@@ -1,199 +1,175 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { GridContainer, Grid } from '@trussworks/react-uswds';
 import { useCollapse } from 'react-collapsed';
+import { LanguageContext } from '../../../contexts/LanguageContext';
 
 import './Jobs.css';
 
-export const Jobs = () => {
+type jobType = {
```

Comment:
Types should star with capital Letter and no need to add "Type" to a `type`. `type Job` should be just fine.